package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "lint": "([ \"$CI\" = true ] && eslint --quiet -f codeframe . || eslint .)",
-    "winlint": "( set CI=true  && eslint --quiet -f codeframe . || eslint .)",
+    "winlint": "(if defined CI (eslint --quiet -f codeframe .) else (eslint .))",
```

Comment:
`if defined` is not correct, in your case if CI=false, then `eslint --quiet -f codeframe .` will executed.